dicom-archive/get_dicom_info.pl

```diff
@@ -160,15 +262,39 @@ sub get_dircos {
     return ($xcos, $ycos, $zcos);
 }
 
-# Routine to convert world coordinates
+=pod
+
+=head3 convert_coordinates(@coords)
+
+Routine to convert world coordinates (multiply them by -1).
```

Comment:
Actually, only the `X` and `Y` part of the coordinates are multiplied by -1 and `Z` is left as is.